kernel/src/vm/vas2.c

```diff
@@ -0,0 +1,24 @@
+#include <vas2.h>
+#include <stdlib.h>
+#include <pmm.h>
+
+struct vas2 * create_vas() {
+    struct vas2 * newvas = kmalloc(sizeof(struct vas2));
+
+    *newvas = (struct vas2) {
+        .tlbDescriptor = request_tlb_descriptor(),
+        .l1PageTable = pmm_allocate_l1_pagetable(),
+    };
+
+    return newvas;
+}
+
+
+void switch_to_vas(struct vas2 * vas) {
+
+}
+
+void free_vas() {
+
```

Comment:
Maybe implement it, or add a unimplemented notice?

Reply:
Adding some VAS methods is a good idea yeah.

Reply:
The latest few commits have added the necessary VAS methods.